src/app/pages/TopicPage/HierarchicalGrid/dataStructures.js

```diff
@@ -0,0 +1,122 @@
+import { css } from '@emotion/react';
+
+const DB = css({
```

Comment:
Think it would really help for readability to rename these consts as you suggest here: https://github.com/bbc/simorgh/pull/10313#discussion_r978537114